index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chimezie's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+            <header class="head-container">
+                <div class="logo">
+                    <h3>Asouzu Chimezie</h3>
+                </div>
+                <div class="menu">
+                    <img src="/images/Union.png" alt="hamburger icon">
+                </div>
+            </header>
+        <section>
+            <div class="main">
+                <div class="wrapper">
+                    <div class="welcome-note">
+                        <h1>I'm Chimezie Glad to see you!</h1>
+                    </div>
+                    <div class="info">
+                        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="socials">
+                        <p>LET'S CONNECT</p>
+                    </div>
+                    <div>
+                        <button><img src="/images/Vector.svg" alt=""></button>
+                        <button><img src="/images/Vector (1).svg" alt=""></button>
+                        <button><img src="/images/Vector (2).svg" alt=""></button>
+                        <button><img src="/images/Vector (3).svg" alt=""></button>
+                        <button><img src="/images/Linkedin icon.svg" alt=""></button>
+                    </div>
```

Comment:
Here you **listed** your social media accounts, right? so don't you think it's better to use `ul` and `li` tags for the **list** of your social media links?

Reply:
Thanks for the feedback i have done the necessary adjustments.